luigi/configuration/toml_parser.py

```diff
@@ -22,6 +22,7 @@
     toml = False
 
 from .base_parser import BaseParser
+from ..freezing import recursively_freeze
```

Comment:
why are we referencing relative paths? can we not `from luigi.configuration.freezing import recursively_freeze`?